nbconvert/exporters/script.py

```diff
@@ -19,15 +18,29 @@ def _template_file_default(self):
 
     def from_notebook_node(self, nb, resources=None, **kw):
         langinfo = nb.metadata.get('language_info', {})
-        
         # delegate to custom exporter, if specified
         exporter_name = langinfo.get('nbconvert_exporter')
         if exporter_name and exporter_name != 'script':
             self.log.debug("Loading script exporter: %s", exporter_name)
-            from .export import get_exporter
+            from .export import get_exporter, ExporterNameError
             if exporter_name not in self._exporters:
-                Exporter = get_exporter(exporter_name)
-                self._exporters[exporter_name] = Exporter(parent=self)
+                try: 
+                    Exporter = get_exporter(exporter_name)
+                    self._exporters[exporter_name] = Exporter(parent=self)
+                except ValueError:
+                    if langinfo.get('nbconvert_exporter_package',""):
+                        external_exporter_name = langinfo.get('nbconvert_exporter_package')
+                        try: 
+                            Exporter = get_exporter(external_exporter_name)
```

Comment:
So for this, I don't feel confident in writing the code that succeeds at this without being able to refer to and use an external script exporter. That means including a test for this functionality. 

If we want to go down this route, is there a way to add a something to the entrypoints.nbconvert.exporters.script only for the purpose of testing?